src/lib.rs

```diff
@@ -536,8 +545,11 @@ impl Adapter {
 
 impl Device {
     /// Check for resource cleanups and mapping callbacks.
-    pub fn poll(&self, force_wait: bool) {
-        wgn::wgpu_device_poll(self.id, force_wait);
+    pub fn poll(&self, maintain: Maintain) {
+        wgn::wgpu_device_poll(self.id, match maintain {
+            Maintain::Poll => false,
```

Comment:
What if we have `Maintain` in wgpu-core under `[repr(C)]` with defined values? bools are never nice anyway

Reply:
This sounds like a good idea. I'll make a pr.